dev/src/lobster/typecheck.h

```diff
@@ -313,6 +330,20 @@ struct TypeChecker {
         return a == b;  // Not inlined for documentation purposes.
     }
 
+    bool TypeOfUndefinedGenericSpec(TypeRef t) {
+        if (!IsUDT(t->t)) return false;
+        return IsUndefinedGenericSpec(t->udt);
+    }
+
+    bool IsUndefinedGenericSpec(UDT* udt) {
+        for (int i = 0; i < udt->generics.size(); i++) {
+            auto& utr = udt->generics[i].giventype.utr;
```

Comment:
use `&` on the right. Same with `*` above and anywhere else. Generally, keep the style of surrounding code.